src/content/vintage/use-the-api/management-api/cluster-apps/_index.md

```diff
@@ -1,15 +0,0 @@
----
-linkTitle: Cluster App Charts
-title: The Cluster App Charts to build new clusters
-description: The Giant Swarm Cluster Charts to provision new clusters and default components.
-weight: 100
-aliases:
-  - /use-the-api/management-api/cluster-apps
-menu:
-  main:
-    identifier: uiapi-cluster-apps
-    parent: uiapi-managementapi
-last_review_date: 2024-02-09
-owner:
-  - https://github.com/orgs/giantswarm/teams/team-horizon
----
```

Comment:
@pipo02mix Is it expected that this file gets removed here?

Reply:
hmm no, I guess the cleaning is not working well. Let me fix that

Reply:
in theory it should filter this one out
`find ${DESTINATION} -type f -not -name "_index.md" | xargs -I '{}' rm '{}'`